### ISSUE TITLE
Incomplete alignments between CINECA & Genomics England

### ISSUE THREAD

**Issue by beckyjackson** [IHCC-cohorts/data-harmonization#51]
While working on mapping Genomics England up to CINECA, I've come across many instances where CINECA does not have a good category for a Genomics England field.

For example, Genomics England has many fields about A&E department diagnoses/ICD-10 codes, which may be either disease or trauma. CINECA only has categories relevant to disease diagnoses.

Another thing that many of these cohorts collect data on is pregnancy and childbirth, which CINECA does not include either.

Maelstrom includes a bit more of the details we need, like "Pregnancy, childbirth and the puerperium" (which is under Diseases?) and "injury, poisoning and certain other consequences of external causes" (also Diseases?). One solution is to put these sorts of things under Diseases, like Maelstrom has.

It may also be easier to map up to Maelstrom first, and then align Maelstrom under CINECA (or align Maelstrom first and then map GE to Maelstrom, I guess the order doesn't matter). The categories of Maelstrom are much more detailed and would be easier to map Genomics England to.

**Comment by jamesaoverton** [comment]
@mcourtot: Since Maelstrom categories are a close match for many GE terms, can we use them for mapping?

**Comment by beckyjackson** [comment]
I mapped Maelstrom to CINECA with some success - there are multiple top-level Maelstrom categories that don't really fall under CINECA, but have been useful for the other cohort mappings:

- Cognition, personality and psychological measures and assessments
- Death
- Health and community care services utilization
- Perception of health, quality of life, development and functional limitations
- Life events, life plans, beliefs and values
- Physical environment
- Preschool, school and work life *
- Social environment and relationships

\* this could fall under socio-demographic, but I think it makes an interesting distinction between occupation (under socio-demographic) and "work life", which has more to do with the day-to-day conditions of work rather than a history of employment.

After completing this mapping, I started to map the other cohorts to Maelstrom instead of CINECA. I had much more coverage using Maelstrom than CINECA, especially with Genomics England. Even the other cohorts had much better mappings (e.g., GCS talks about tea and animals, which fall under lifestyle information. CINECA has lifestyle and behaviours, but Maelstrom's is much more complete.

I think continuing to map to Maelstrom rather than CINECA might be easiest for future cohorts to do their own mappings with, but I'd like to hear others' thoughts as well. The one thing that Maelstrom doesn't specify is the "biosample type" which seems to be pretty important for the browser.
